guide/english/machine-learning/deep-learning/index.md

```diff
@@ -42,6 +42,10 @@ These days there are a variety of deep learning frameworks that allow you specif
 * <a href="https://github.com/gluon-api/gluon-api/">Gluon</a>
 * <a href="http://deeplearning.net/software/theano/">Theano</a>
 
+### Deep learning in Android & Web  
```

Comment:
Please take these changes into PR, it seems to be out of scope of the PR

Reply:
Oops. I didn't touch this file. Just remove it?

Reply:
Yup, please remove it

Reply:
@ShepherdNg Don't delete the file.  Just revert the changes for it.